doc/user-guide.md

```diff
@@ -111,10 +110,10 @@ Each `OpenLibertyApplication` CR must specify `applicationImage` parameter. Spec
 | `route.insecureEdgeTerminationPolicy`   | HTTP traffic policy with TLS enabled. Can be one of `Allow`, `Redirect` and `None`. |
 | `route.certificate`  | A YAML object representing a [Certificate](https://cert-manager.io/docs/reference/api-docs/#cert-manager.io/v1alpha2.CertificateSpec). |
 | `route.certificateSecretRef` | A name of a secret that already contains TLS key, certificate and CA to be used in the route. Also can contain destination CA certificate.  |
-| `sso`   | Specifies the configuration for single sign-on providers to authenticate with. Specify the sensitive information such as _clientId_  and _clientSecret_ for the selected providers using the Secret. |
-| `sso.mapToUserRegistry`   | Specifies whether to map user identifier to registry user. Applies to all providers. Default value is _false_ |
-| `sso.redirectToRPHostAndPort`   | Specifies a callback host and port number. Applies to all login providers. |
-| `sso.github.hostname`   | The hostname of GitHub. Needed for Github Enterprise (for example: github.mycompany.com). Default value is _github.com_ |
+| `sso`   | Specifies the configuration for single sign-on providers to authenticate with. Specify sensitive information such as _clientId_  and _clientSecret_ for the selected providers using a Secret. See [Single Sign-On (SSO)](#single-sign-on-sso) for more info. |
```

Comment:
@chirp1 In addition to changed content from this PR. Could you please also review rest of the parameters for `sso` from this table?